tests/cli/test_cli_orchestration.py

```
@@ -28,7 +29,8 @@
 class FakePopulationBackend:
     """Minimal public-style population backend for orchestration tests."""
 
-    parameter_names = ("detector_frame_mass_1", "detector_frame_mass_2", "coa_time")
+    parameter_names: ClassVar[tuple[str, ...]] = ("detector_frame_mass_1", "detector_frame_mass_2", "coa_time")
+    metadata: ClassVar[dict[str, object]] = {"fetch": {"scheme": "https"}, "resolved_path": "/tmp/catalog.h5"}
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/catalog.h5"

(S108)